packages/list-store/src/search-params.ts

```diff
@@ -55,11 +59,30 @@ export function getUrlWithSearchParams({
     });
   }
 
-  // Only add relevant values to the search params. Remove all keys with empty strings as values
-  Object.keys(searchParams).forEach(key =>
-    searchParams[key] === '' ? delete searchParams[key] : {}
-  );
-  const encodedSearchParams = new URLSearchParams(searchParams).toString();
+  // Only add relevant values to the search params.
```

Comment:
Very minor: the comment suggests that relevant values are going to be added. But lines 64-66 do not add values, only remove. Rephrase the comment?